### DISCUSSION TITLE
Question: How to share resources like SolidColorBrush between a controls library and my app?

### DISCUSSION THREAD

**Discussion by HppZ** [microsoft/WindowsAppSDK#1289] [Q&A] [answered]
How to share a SolidColorBrush (one instance, need to be accessible by default control style, named control style and my main app) between a controls library and my app?

**Comment by btueffers** [comment, marked as answer]
Hi @HppZ , thank you for participating in the Windows App SDK community! As this topic is related specifically to WinUI we ask that you open an issue in the [WinUI repo](https://github.com/microsoft/microsoft-ui-xaml/issues). Guidance to do so is now included in the [Windows App SDK contributor guide](https://github.com/microsoft/WindowsAppSDK/blob/main/docs/contributor-guide.md)!
